include/vctr/Containers/VctrBase.h

```diff
@@ -390,18 +394,55 @@ class VctrBase : private Config,
     }
 
     /** Returns the number of elements that are equal to valueToLookFor. */
-    constexpr size_t count (const ElementType& valueToLookFor)
+    constexpr size_t count (const ElementType& valueToLookFor) const
     {
         return std::count (begin(), end(), valueToLookFor);
     }
 
     /** Returns the number of elements that satisfy predicate. */
-    template <is::elementPredicateFunction<ElementType> Fn>
-    constexpr size_t countIf (Fn&& predicate)
+    template <is::functionWithSignatureOrImplicitlyConvertible<bool (const ElementType&)> Fn>
+    constexpr size_t countIf (Fn&& predicate) const
     {
         return std::count_if (begin(), end(), std::forward<Fn> (predicate));
     }
 
+    /** Returns true if all elements satisfy the predicate or if it is empty */
```

Comment:
Maybe "… or if container is empty", because "it" might be misunderstood as referring to "the predicate".

Reply:
good point